.travis.yml

```diff
@@ -11,6 +11,10 @@ rvm:
 services:
   - postgresql
   - elasticsearch
+addons:
+  chrome: beta
+before_install:
+  - "/Applications/Google\\ Chrome.app/Contents/MacOS/Google\\ Chrome --headless --disable-gpu --remote-debugging-port=9222 http://localhost &"
 before_script:
```

Comment:
Can you have an environment variable in there to avoid the MacOS-specific stuff?

Reply:
Ok will look into this